### DISCUSSION TITLE
Start publishing cannot be executed - Secured connection to AWS IoT core using TPM2.0

### DISCUSSION THREAD

**Discussion by tkaehad** [Infineon/optiga-tpm-explorer#10] [Q&A]
I am demonstrating "Secured connection to AWS IoT core using TPM2.0".

Following the steps, I have successfully executed the 1-click provision and have started to subscribe messages in AWS IoT.
Upon executing Start publishing, I received the following error message.

```
sudo ../eHealthDevice -m "SPO2: 96, BPM: 60" -t pulsioximeter -e ....

sudo: unable to execute ../eHealthDevice: No such file or directory
```

If the "eHealthDevice" path is incorrect, what path should I put this in?

Please advise how to resolve this issue.

**Comment by RaymWong** [comment]
> I am demonstrating "Secured connection to AWS IoT core using TPM2.0".
> 
> Following the steps, I have successfully executed the 1-click provision and have started to subscribe messages in AWS IoT. Upon executing Start publishing, I received the following error message.
> 
> ```
> sudo ../eHealthDevice -m "SPO2: 96, BPM: 60" -t pulsioximeter -e ....
> 
> sudo: unable to execute ../eHealthDevice: No such file or directory
> ```
> 
> If the "eHealthDevice" path is incorrect, what path should I put this in?
> 
> Please advise how to resolve this issue.

Hi,
Thanks for the feedback.
First please ensure that the "eHealthDevice" executable file is in the repo directory:  ~/optiga-tpm-explorer/Python_TPM20_GUI/bin/
In the "bin" folder, and run the "start_gui.sh".  See if this helps. Thanks
